### ISSUE TITLE
Create actions section and Quick Deploy button in Deployment Card

### ISSUE THREAD

**Issue by dotNomad** [posit-dev/publisher#931]
Part of the work coming from https://github.com/posit-dev/publisher/issues/895

We want to add a "Quick Deploy" button to the Deployment Card. We need an actions section and the actual button itself.

We need to support screen readers so the icons should have aria labels. The label should change if it is a deployment or re-deployment.

The quick deploy button should disabled when in progress or the deployment cannot be parsed.
